public/css/main.css

```diff
@@ -1,29 +1,78 @@
+* {
+  margin:0;
+  padding:0;
+  box-sizing:border-box;
+}
+body {
+  background-color: #F2F2F2;
+}
 a, a:hover, a:active, a:visited, a:focus {
     text-decoration: none;
 }
-.calendar:hover {
-  color: #6495ED;
+.header {
+  background-color: #409A93;
+  font-family: 'Noto Sans', sans-serif;
+  padding: 1.5em;
+  text-align: center;
+}
+.naz-events{
+  color: white;
+  font-weight: bold;
+  font-size: 2em;
 }
 .calendar {
-  color: gray;
+  width: 1.4em;
 }
-.header {
-  background-color:	#F5F5F5;
-  padding: 3px;
-  padding-bottom: 12px;
-  box-shadow: 4px 3px 12px;
-  font-family: 'Quattrocento Sans', sans-serif;
+.calendar:hover {
+  color: white;
 }
-.naz-events{
-  color: #6495ED;
+.div-calContainer{
+  float: right;
+  width: 10%;
+  margin-top: 0.7em;
 }
-.div-calendarBtn{
-  margin-top: -37px;
-  margin-right: 7px;
-  direction: rtl;
+.plusIcon{
+  width: 1.4em;
 }
-span.glyphicon {
-  font-size: 20px;
+.plusIcon:hover {
+  color:white;
+}
+.div-plusContainer{
+  float: left;
+  width: 10%;
+  margin-top: 0.7em;
+}
+.languageNav {
+  background: #F2F2F2;
+  width: 100%;
+  padding: 0.4em;
+  margin-bottom: 0.4em;
+  box-shadow: 0.2em 0.3em 0.3em #DBDBDB;
+}
+.engBtn{
+  width: 48%;
+  float: right;
+  margin-right: 0.2em;
+  font-weight: bold;
+  font-size: 1.2em;
+  color: #5C5C5C;
+  padding: 0.2em;
+  background: #DBDBDB;
+  border: 0.1em solid #DBDBDB;
+  border-radius: 0.1em;
+  font-family: 'Lato', sans-serif;
+}
+.arBtn{
+  width: 48%;
+  margin-left: 0.2em;
+  font-weight: bold;
+  font-size: 1.2em;
+  color: #5C5C5C;
+  padding: 0.2em;
+  background: #DBDBDB;
+  border: 0.1em solid #DBDBDB;
+  border-radius: 0.1em;
+  font-family: 'Lato', sans-serif;
 }
```

Comment:
`engBtn` and `arBtn` are both the same - can just make one class instead

Also it's still not quite rendering correctly - the spacing around the buttons should be identical on all sides, and between the two buttons. Sorry to be picky but it's something that's very noticeable if it's off slightly.

You can see the issue [here](https://imgur.com/a/NzFx9) - the gap to the top and bottom of the buttons is different to the left and right, and very clearly so in between the buttons too